src/main/java/com/github/ozsie/CheckMojo.kt

```diff
@@ -12,14 +12,18 @@ import java.nio.file.Paths
 @Suppress("unused")
 @Mojo(name = "check",
         defaultPhase = LifecyclePhase.VERIFY,
+        threadSafe = true,
```

Comment:
@Ozsie 
@arturbosch
I take your word that it's threadsafe -
I dug a little into the code but didn't get far
so I base my assumption on https://github.com/Ozsie/detekt-maven-plugin/issues/34

I ran it in parallel on my maven project, and it worked fine.
(`mvn detekt:check -T 4 `)